Foundation/NSOperation.swift

```diff
@@ -47,7 +47,6 @@ open class Operation : NSObject {
 #endif
     }
     
-    /// - Note: Operations that are asynchronous from the execution of the operation queue itself are not supported since there is no KVO to trigger the finish.
     open func start() {
```

Comment:
Any updates on this? The limitation @phausler identifies seems less serious than the current one.